PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1330,18 +1381,22 @@ def ak4JetReclustering(self,process, pfCandCollection, patMetModuleSequence, pos
             if not hasattr(process, "pfCHS"+postfix):
                 setattr(process,"pfCHS"+postfix,pfCHS)
             pfCandColl = cms.InputTag("pfCHS"+postfix)
-            
+                   
 
         jetColName+=postfix
         if not hasattr(process, jetColName):
             process.load("RecoJets.JetProducers.ak4PFJets_cfi")
             
             #if chs:
             setattr(process, jetColName, getattr(process,"ak4PFJets").clone() )
-
             getattr(process, jetColName).src = pfCandColl 
             getattr(process, jetColName).doAreaFastjet = True
             
+            #puppi
+            #if "Puppi" in postfix:
+            #    getattr(process, jetColName).doAreaFastjet = cms.bool(True)
+            #    getattr(process, jetColName).src = cms.InputTag("puppiNoLep")
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant